File: sdk/go/commands.go
package wlteopenapi

import "context"

type CommandsService struct {
	client *Client
}

func (s *CommandsService) GetResult(ctx context.Context, commandID string) (CommandResult, error) {
	var result CommandResult
	err := s.client.request(ctx, "GET", "/wlte/v1/commands/"+urlEscape(commandID), nil, nil, nil, &result)
	return result, err
}

File: sdk/typescript/src/commands.ts
import type { WlteClient } from './client.js'
import type { ApiEnvelope, CommandResult } from './types.js'

export class CommandsApi {
  constructor(private readonly client: WlteClient) {}

  async getResult(commandId: string): Promise<CommandResult> {
    const response = await this.client.request<ApiEnvelope<CommandResult>>(
      `/wlte/v1/commands/${encodeURIComponent(commandId)}`,
    )
    return response.data
  }
}
